config/jobs/kubernetes/sig-azure/sig-azure-config.yaml

```diff
@@ -0,0 +1,81 @@
+presets:
+- labels:
+    preset-azure-cred: "true"
+  env:
+  - name: AZURE_CREDENTIALS
+    value: /etc/azure-cred/credentials
+  - name: AZURE_SSH_PUBLIC_KEY_FILE
+    value: /etc/azure-ssh/azure-ssh-pub
+  - name: REGISTRY
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_USERNAME
+    value: ritazh # update this after we have another account to use
+  - name: DOCKER_PASSWORD_FILE
+    value: /etc/docker-cred/dockercred
+  volumes:
+  - name: azure-cred
+    secret:
+      secretName: azure-cred
+  - name: azure-ssh
+    secret:
+      secretName: azure-ssh
+  - name: docker-cred
+    secret:
+      secretName: docker-cred
+  volumeMounts:
+  - name: azure-cred
+    mountPath: /etc/azure-cred
+    readOnly: true
+  - name: azure-ssh
+    mountPath: /etc/azure-ssh
+    readOnly: true
+  - name: docker-cred
+    mountPath: /etc/docker-cred
+    readOnly: true
+
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-azure
+    always_run: true
```

Comment:
this should be false or `skip_report: true`. when we introduce a new job we don't have it report to PRs at first or require that it be manually requested, otherwise if this job doesn't work right it will block PRs from merging.